core/models.py

```diff
@@ -0,0 +1,125 @@
+import hashlib
+import re
+from typing import ClassVar
+
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+User = get_user_model()
+
+
+class GenericModel(models.Model):
+    user = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    is_active = models.BooleanField(default=True)
+    is_locked = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:  # noqa: D106

```

Comment:
Remove this and add to ruff config in pyproject.toml 
https://github.com/pymasterspl/reddit/blob/494c9c9a743d0090c3eff5ebd50df9af91a2f682/pyproject.toml#L32